environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/response/DetailedEnvironmentResponse.java

```diff
@@ -70,6 +70,8 @@ public void setCredential(CredentialResponse credential) {
 
         private AwsEnvironmentParameters aws;
 
+        private String parentEnvironmentCrn;
```

Comment:
I think it should be great to show the name as well

Reply:
Done in af20f701d8c3e87addc57f612c9b6fc642d0daf5